src/pages/instances/forms/DuplicateInstanceForm.tsx

```diff
@@ -0,0 +1,240 @@
+    queryKey: [queryKeys.cluster],
+    queryFn: fetchClusterMembers,
+  });
+
+  const notifySuccess = (instanceName: string) => {
+    const newInstance = (
+      <Link to={`/ui/project/${instance.project}/${instanceName}`}>
+        {instanceName}
+      </Link>
+    );
+    toastNotify.success(
+      <>
+        Created and started cloned instance <b>{newInstance}</b>.
+      </>,
+    );
+  };
+
+  const formik = useFormik<DuplicateInstanceFormValues>({
+    initialValues: {
+      instanceName: instance.name + "-1",
+      project: "default",
+      allowInconsistent: false,
+      instanceOnly: false,
+      stateful: false,
+    },
+    validationSchema: Yup.object().shape({
+      instanceName: Yup.string().required(),
+    }),
+    onSubmit: (values) => {
+      duplicateInstance(values, instance)
+        .then((operation) => {
+          toastNotify.info(
+            <>
+              Creation for instance <b>{values.instanceName}</b> started.
+            </>,
+          );
+          close();
+
+          eventQueue.set(
+            operation.metadata.id,
+            async () => {
+              //Start Instance if instance created successfully.
+              const instanceToStart = fetchInstance(
+                values.instanceName,
+                values.project,
+              );
+
+              //Display starting notification
+              toastNotify.info(
+                <>
+                  Created instance <b>{values.instanceName}</b>, now starting
+                  it.
+                </>,
+              );
+
+              startInstance(await instanceToStart)
+                .then(() => {
+                  //If instance started successfully, display success message as shown in design.
+                  notifySuccess(values.instanceName);
+                })
+                .catch((e) => {
+                  toastNotify.failure(
+                    `Unable to start instance "${values.instanceName}`,
+                    e,
+                  );
+                });
+            },
+            (msg) => {
+              toastNotify.failure(
+                `Instance creation for "${values.instanceName}" failed.`,
+                new Error(msg),
+              );
+            },
+          );
+        })
+        .catch((e) => {
+          toastNotify.failure(
+            `Instance creation for "${values.instanceName}" failed.`,
+            e,
+          );
+          close();
+        });
+    },
+  });
+
+  return (
+    <Modal
+      close={close}
+      title="Duplicate Instance"
+      buttonRow={
+        <>
+          <Button
+            appearance="base"
+            className="u-no-margin--bottom"
+            type="button"
+            onClick={close}
+          >
+            Cancel
+          </Button>
+          <ActionButton
+            appearance="positive"
+            className="u-no-margin--bottom"
+            loading={formik.isSubmitting}
+            disabled={!formik.isValid}
```

Comment:
let's also disable the submit button if data is still loading for any one of the selectors.